src/read.c

```diff
@@ -6380,8 +6431,11 @@ avifResult avifDecoderReset(avifDecoder * decoder)
         decoder->image->transformFlags |= AVIF_TRANSFORM_IMIR;
         decoder->image->imir = imirProp->u.imir;
     }
+    if (alphaProperties) {
```

Comment:
Nit: Change `gainMapProperties != NULL` (line 6437) to `gainMapProperties` for consistency.